src/Workspaces/CoreTest/ServicesTest.csproj

```diff
@@ -49,6 +51,12 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="$(OutputPath)\..\..\Exes\csc\net46\csc.exe">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="$(OutputPath)\..\..\Exes\vbc\net46\vbc.exe">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
Can you explain why this is needed? Copying the exes alone isn't generally useful due to issues like binding redirects.

Reply:
It's all horrible hackery to try and get the targets and props in the Microsoft.Build.Runtime packages. I've decided to take a completely different approach. I'll chat with you about it a bit later.